embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/EmbraceExtensions.kt

```diff
@@ -192,3 +193,7 @@ internal fun io.embrace.android.embracesdk.Severity.toOtelSeverity(): Severity =
     io.embrace.android.embracesdk.Severity.WARNING -> Severity.WARN
     io.embrace.android.embracesdk.Severity.ERROR -> Severity.ERROR
 }
+
+internal fun String.isValidLongValueAttribute() = longValueAttributes.contains(this)
```

Comment:
This is needed to bypass the attribute value length that we set to prevent really long attributes from being added so that we can store stacktraces, which will almost always be pretty big, so I'm bypassing this limit explicitly here. Other attributes could be allow-listed thusly, though we may want to revisit how this is implemented if we have more than a handful.